lib/services/firewall.pm

```diff
@@ -30,13 +30,13 @@ sub install_service {
 }
 
 sub susefirewall2_to_firewalld {
-    my $timeout = 180;
+    my $timeout = 360;
     assert_script_run('susefirewall2-to-firewalld -c',                                     timeout => $timeout);
```

Comment:
Wei, 

Can you pls add some comments why you add $timeout variable here, and why you use 360s?

Reply:
I have not added $timeout variable , this is original code, i just update the timeout value:)

Reply:
Why 360?
Timeout of susefirewall2-to-firewalld will cost 280s (after check openqa result), so i think 360s currently maybe a good value we can have a try.

Reply:
Yup, then no problem here.

I'd suggest adding some comment here.